scrapers/FemdomEmpire.yml

```diff
@@ -0,0 +1,63 @@
+name: "FemdomEmpire"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/trailers/
+      - feminized.com/tour/trailers/
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/models/
+      - feminized.com/tour/models/
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="videoDetails clear"]/h3
+      Date:
+        selector: //p[span[contains(.,"Date Added")]]
+        postProcess:
+          - replace:
+            - regex: "Date Added:"
+              with:
+          - parseDate: January 2, 2006
+      Details: //div[@class="videoDetails clear"]/p
+      Performers:
+        Name: //li[@class="update_models"]/a
+      Tags:
+        Name: //ul/li[@class="label" and contains(.,"Categories:")]/..//a
+      Studio:
+        Name:
+          selector: //div[@id='logo']/h1/a/span
+        Image:
+          selector: //img[@class="update_thumb thumbs stdimage"]/@src0_1x
+          postProcess:
+            - replace:
+              - regex: .*/tour/content//
+                with: "https://feminized.com/tour/content/"
+  performerScraper:
+    common:
+      $performerInfo: "//div[@class='contentBox'][1]"
+      # if stash scraper supports xpath 2 we might be able to parse measurements etc from femdomempire stats lists.
+      # $profileAbout: "$performerInfo/div[@class='profile-about']/p"
```

Comment:
I wouldnt even bother with how the tags are inserted in the stats list.
I would also remove the comment

Reply:
fair enough. I've no real interest in the measurements thing anyway.